src/include/rados/librados.hpp

```diff
@@ -1019,8 +1019,6 @@ namespace librados
     void set_assert_version(uint64_t ver);
     void set_assert_src_version(const std::string& o, uint64_t ver);
 
-    const std::string& get_pool_name() const;
-
```

Comment:
Yeah, this is annoying... I guess we need to keep the member var after all, just for return values for this version